Update safety.md (#3491)
tigerbeetle/tigerbeetle: docs/concepts/safety.md

# Safety

The purpose of a database is to store data: if the database accepts new data, it should be able to
retrieve it later. Surprisingly, many databases don't provide guaranteed durability -- usually the
data is there, but, under certain edge case conditions, it can get lost!

As the purpose of TigerBeetle is to be the system of record for business transaction, associated
with real-world value transfers, it is paramount that the data stored in TigerBeetle is safe.

> TigerBeetle is therefore designed, engineered, and tested to deliver unbreakable durability —- 

Reviewer: ```suggestion
TigerBeetle is therefore designed, engineered, and tested to deliver unbreakable durability -- 
```

different cloud providers (two replicas per provider). Because TigerBeetle uses
[Heidi Howard's flexible quorums](https://arxiv.org/pdf/1608.06696v1), this deployment is guaranteed
to tolerate a complete outage of any cloud provider and will likely survive even if one extra

> replica fails. Multi-cloud eliminates lock-in, meets regulatory requirements, and protectsavailability 

Reviewer: ```suggestion
replica fails. Multi-cloud eliminates lock-in, meets regulatory requirements, and protects availability 
```

> -— even through provider slowdowns and disruptions.

Reviewer: ```suggestion
-- even through provider slowdowns and disruptions.
```

### Durability

> Without Durability, the guarantees of Atomicity, Consistency, and Isolation collapse -- the only 

Reviewer: ```suggestion
Without Durability, the guarantees of Atomicity, Consistency, and Isolation collapse -- the only 
```
